client/actions/userActions.js

```diff
@@ -9,7 +10,8 @@ import { handleError } from '../utils/errorHandler';
 
 export const signOut = () => (dispatch) => {
   localStorage.removeItem('token');
-  location = '/login';
+  Cookies.remove('jwt-token');
```

Comment:
Great job on this fix @ipheghe. I think we should not remove this cookie though. It is set by the `andela.com` domain and applies to all subdomains. Removing it will also log the user out of all their Andela applications, which is not desirable. What we can do instead is to remove copying from cookies to localstorage in `client/pages/Dashboard/index.jsx: line 144`.